beetsplug/playlist.py

```diff
@@ -23,6 +22,10 @@
 from beets.util import path_as_posix
 
 
+def is_m3u_file(path):
+    return os.path.splitext(path)[1].lower() in {".m3u", ".m3u8"}
```

Comment:
Use `pathlib.Path` and add types

```suggestion
def is_m3u_file(path: str) -> bool:
    return Path(path).suffix.lower() in {".m3u", ".m3u8"}
```